### PR TITLE
✨ Debugger displaying multiple experiences

### PR DESCRIPTION
**Pull request by andretortolano**
adding debugger ability to read multiple experiences.

suggestion to add the frame id when it exists.

image

### CODE REVIEW

**Review by iujames — APPROVED** [review]
nice, i like the idea of showing the frame ID in the debugger. Adding to the actual analytics data seems harmless, it would just be another field that gets put into Snowflake afaik. The only other way I could think to do that would be to scan the StateMachines that are currently in the directory, and look at the current experience in each one to find any that match the ID, then grab the render context / frame ID from there. fyi to @mmaatttt on the topic of adding frame ID, for consistency.

File: appcues/src/main/java/com/appcues/analytics/ExperienceLifecycleEvent.kt [hunk 1]
```diff
@@ -85,6 +86,7 @@ internal sealed class ExperienceLifecycleEvent(
             "experienceId" to experience.id.appcuesFormatted(),
             "experienceName" to experience.name,
             "experienceType" to experience.type,
+            "frameId" to experience.renderContext.getFrameId(),
```
Comment (mmaatttt) [inline appcues/src/main/java/com/appcues/analytics/ExperienceLifecycleEvent.kt:89]: Is this an empty string for a modal?

Comment (iujames) [reply appcues/src/main/java/com/appcues/analytics/ExperienceLifecycleEvent.kt:89]: looks like it is. good catch - ideally a null / trimmed out

Comment (andretortolano) [reply appcues/src/main/java/com/appcues/analytics/ExperienceLifecycleEvent.kt:89]: its null. is that ok?

**Pull request merged**